File: projectapi/Models/Enquiry.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace projectapi.Models
{
    public class Enquiry
    {
        public int EnquiryID { get; set; }
 
        public DateTime EnquiryDate { get; set; }
 
        public string? Title { get; set; }
 
        public string? Description { get; set; }
 
        public string? EnquiryType { get; set; }
 
        public int? UserId { get; set; }
        public string? CreatedBy {get; set;}
        public DateTime? CreatedAt {get;set;}
        public string? ModifiedBy {get; set;}
        public DateTime? ModifiedAt {get;set;}
 
        public Boolean? IsDeleted { get; set; }
        public virtual User? User {get;set;}
    }
}

File: angularapp/src/app/Models/Enquiry.ts
import { Course } from "./Course";
 
export interface Enquiry {
    createdBy: string,
    userId: number,
    createdAt:Date ,
    enquiryType: string,
    modifiedBy: null,
    description: string,
    modifiedAt: Date,
    title:string,
    isDeleted: Boolean,
    enquiryDate: Date,
    //user:User,
    enquiryID: number,
    course:Course
    courseName:string
}